consensus/istanbul/core/commit.go

```diff
@@ -137,6 +138,7 @@ func (c *core) broadcastCommit(sub *istanbul.Subject) {
 }
 
 func (c *core) handleCommit(msg *istanbul.Message) error {
+	defer func(start time.Time) { c.handleCommitTimer.UpdateSince(start) }(time.Now())
```

Comment:
you don't need the xtra funciton
```suggestion
	defer c.handleCommitTimer.UpdateSince(time.Now())
```

Reply:
Nice!